sql/catalyst/src/main/scala/org/apache/spark/sql/types/CalendarIntervalType.scala

```diff
@@ -22,11 +22,14 @@ import org.apache.spark.unsafe.types.CalendarInterval
 
 /**
  * The data type representing calendar time intervals. The calendar time interval is stored
- * internally in two components: number of months the number of microseconds.
+ * internally in three components:
+ *   the number of months
```

Comment:
I think we should say explicitly that the numbers can be positive as well as negative